ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```
@@ -6505,4 +6505,15 @@ public StorageHandlerInfo getStorageHandlerInfo(Table table)
       throw new HiveException(e);
     }
   }
+
+  public void alterTableExecuteOperation(Table table, AlterTableExecuteSpec executeSpec) throws HiveException {
+    try {
+      HiveStorageHandler storageHandler = createStorageHandler(table.getTTable());
+      if (storageHandler.supportsExecuteOperations()) {
```

Shall we throw an exception if the feature is not supported by the table?